### PR TITLE
Add Support for BF.INSERT

### PR DESCRIPTION
**Pull request by dengliming**
for [https://github.com/RedisBloom/redisbloom-go/issues/6](https://github.com/RedisBloom/redisbloom-go/issues/6)

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -176,7 +176,21 @@ func (client *Client) BfInsert(key string, cap int64, errorRatio float64, expans
 		args = args.Add("NONSCALING")
 	}
 	args = args.Add("ITEMS").AddFlat(items)
-	return redis.Int64s(conn.Do("BF.INSERT", args...))
+	var resp []interface{}
+	var innerRes int64
+	resp, err = redis.Values(conn.Do("BF.INSERT", args...))
```
[the remark this answers is not in the document]
Comment (dengliming) [reply client.go:181]: Looks great. But I'm still a little confused. Did I misunderstand the document？I found that filter command document is different.
Bloom_Commands
```
Returns ¶
An array of booleans (integers). Each element is either true or false depending on whether the corresponding input element was newly added to the filter or may have previously existed.
```
Cuckoo_Commands
```
Returns ¶
An array of booleans (as integers) corresponding to the items specified. Possible values for each element are:

> 0 if the item was successfully inserted
0 if the item already existed and INSERTNX is used.
<0 if an error ocurred
Note that for CF.INSERT , unless an error occurred, the return value will always be an array of >0 values.
```
Maybe The server side should handle this exception uniformly？like return -1 if an error ocurred. :）

[the remark this answers is not in the document]
Comment (filipecosta90) [reply client.go:181]: @dengliming to be backwards compatible we can't change the reply format on RedisBloom.
Regarding the docs you're right. We're missing the description of what happens when an error occurs :)

[the remark this answers is not in the document]
Comment (filipecosta90) [reply client.go:181]: I will merge this PR, given that the client is working as expected and open a PR to extend/clarify the docs on RedisBloom. :+1:

[the remark this answers is not in the document]
Comment (dengliming) [reply client.go:181]: @filipecosta90 👍

**Pull request merged**